.azure-pipelines/linux-conda-CI.yml

```diff
@@ -42,6 +42,10 @@ jobs:
       pip install tensorflow
       pip install $(KERAS)
       pip install $(TF2ONNX)
+      git clone http://github.com/onnx/onnxmltools
```

Comment:
Shall we change this to pip install after the package is published?

Reply:
Thanks for the review, @jiafatom! Yes, this PR is waiting for the package to be published.

Reply:
Resolved in latest update! Using pypi package: https://pypi.org/project/onnxconverter-common